### PR TITLE
Frontend quiz creation implementation

### PR DESCRIPTION
**Pull request by risakawagoe**
MVP quiz features for creating quiz with

- multiple choice
- multiple select
- short answer
- no asnwer

questions. Button "Create" logs the JSON object to be sent to the backend.

### CODE REVIEW

File: frontend/src/components/quizzerhub_components/quiz_answer_form.tsx
Comment (toastjpg) [inline frontend/src/components/quizzerhub_components/quiz_answer_form.tsx:1]: is this file left commented out intentionally?

**Review by toastjpg — APPROVED** [review]
LGTM

Nice frontend overhaul. I assume linking to backend service for storage of quiz a WIP still

File: frontend/src/components/quizzerhub_components/quiz_answer_form.tsx
Comment (risakawagoe) [reply frontend/src/components/quizzerhub_components/quiz_answer_form.tsx:1]: Mb i forgot to delete it. In the version I have under branch (quiz-service) I have deleted it as it is no longer needed.

**Comment by risakawagoe** [comment]
Regarding "linking to backend service for storage" I have left my updates on discord:)

**Pull request merged**